webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/web/services/JsTableData.scala

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Have you deleted the original file in rudder-web ? it should have been a git mv command ! just rm JsTableData.scala file in rudder-web it should link it